test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class GamesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get games_index_url
+    assert_response :success
+  end
+
+  test "should get –skip-routes" do
```

Comment:
This test seems to be failing for me @amykingOW. Are you just wanting to test that we can ignore some routes?

Reply:
I believe there was a typo, fixed in https://github.com/amykingOW/hangman/pull/1/commits/6dea97db8150c9b36f1c9a986b85b02e9caa964d. However, I'm not really fussed about testing right now. This PR was just showing a welcome page.